### PR TITLE
chore(testing): update protractor to version 2.1.0

### PR DESCRIPTION
**Pull request by juliemr**

### CODE REVIEW

**Comment by juliemr** [comment]
This is currently failing because Protractor 2.1.0 upgrades to Chromedriver 2.15 (from Chromedriver 2.14). Chromedriver 2.15 no longer works with old versions of Chrome, and Dartium has an old version number. So it errors with `Chrome version must be >= 40.0.2214.0`.

Dartium probably won't be upgraded for a while, so we'll have to either hold off on the upgrade or download a custom version of Chromedriver when dealing with dartium.

**Comment by juliemr** [comment]
Modified with a custom script to make sure we use Chromedriver@2.14 until Dartium is  updated.

File: npm-shrinkwrap.clean.json [hunk 1]
```diff
@@ -8606,392 +8440,6 @@
             }
           }
         },
-        "tsd": {
```
Comment (tbosch) [inline npm-shrinkwrap.clean.json]: Why did this change?

Comment (juliemr) [inline npm-shrinkwrap.clean.json]: I think these were preexisting changes that never got put into shrinkwrap. @vicb tried it out and got the same effect on his machine.

**Comment by juliemr** [comment]
Thanks! Rebased and merged as dd9b08cce8b1e435f052184d8d46b8a5381ff0f3

**Pull request closed without merging**